.circleci/config.yml

```diff
@@ -55,6 +56,30 @@ restore_lib: &restore_lib
     # Restore from the most recently cached version of lib, using a partial key match
     key: v3-lib-cache-
 
+hub_deploy: &hub_deploy
+  after_build:
+    - run: heroku container:login
+    - run: docker push registry.heroku.com/simple-hub-staging/simple-hub
+    - run: heroku container:release -a simple-hub-staging simple-hub
+  before_build:
+    - attach_workspace:
+        at: /home/circleci/project
+    - run: curl https://cli-assets.heroku.com/install.sh | sh
+    - run: cd packages/simple-hub && yarn build
+  deploy: false
+  dockerfile: packages/simple-hub/docker/simple-hub.dockerfile
+  docker-password: HEROKU_API_KEY
+  docker-username: HEROKU_USERNAME
```

Comment:
What's this? Also why aren't these prefixed with a `$`?

Reply:
https://circleci.com/orbs/registry/orb/circleci/docker#executors-docker: Name of environment variable storing your Docker password